Makefile

```
@@ -1,7 +1,13 @@
-prepare:
-	python3.11 -m venv env
-	env/bin/python -m pip install robotframework
+venv:
+	python3.11 -m venv .venv
+	python -m pip install robotframework
+	python -m pip install requests
+	python -m pip uninstall robot
+
+clean-venv:
+	deactivate
+	rm -rf .venv
```

This is what we use in the other Python projects to keep it the same.

```suggestion
clean_venv: ## Remove virtual environment
	@echo "Cleaning venv"
	@rm -rf .venv
```